### PR TITLE
Add go strings

### LINKED ISSUE #421: Add Go strings
# Description

Strings for Go aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/strings.json` to the right Go directory in `web/thesauruses/`
* Edit the strings.json file to accurately reflect the latest major version of Go
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then delete the `"code"` line.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by 2kindsofcs**
## What GitHub issue does this PR apply to?

Resolves #421

## What changed and why?

Added go strings.

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/go/strings.json [hunk 1]
```diff
@@ -0,0 +1,287 @@
+{
+  "meta": {
+    "language": "language_id",
+    "language_version": "version.number",
+    "language_name": "Human-Friendly Language Name"
```
Comment (geekygirlsarah) [inline web/thesauruses/go/strings.json]: Can you update these to reflect Go and the version number you're using?

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
Just make that minor change and this should be good to merge in!

File: web/thesauruses/go/strings.json (hunk 1 above)
Comment (2kindsofcs) [reply web/thesauruses/go/strings.json]: Of course. I updated it.

**Review by geekygirlsarah — APPROVED** [review]

**Comment by geekygirlsarah** [comment]
You force-pushed, so it couldn't show me your changes since last time, but re-reviewing it looked fine to me.

I'll merge it in. Thank you!

**Pull request merged**